hudi-common/src/main/java/org/apache/hudi/common/model/HoodieWriteStat.java

```diff
@@ -159,6 +159,9 @@ public class HoodieWriteStat implements Serializable {
   @Nullable
   private Long maxEventTime;
 
+  @Nullable
+  private String prevBaseFile;
```

Comment:
so you just need the basefile? not the entire previous file slice?

Reply:
HoodieDeltaWriteStat contains the log files. So, HoodieWriteStat only contains info about base files.